src/index.ts

```diff
@@ -23,12 +24,8 @@ class MyClient extends AkairoClient {
 export const client = new MyClient();
 
 const options: ConnectionOptions = {
+  url: DATABASE_URL,
```

Comment:
We should keep ORM configuration out of code, in config file: https://github.com/typeorm/typeorm/blob/master/docs/using-ormconfig.md